packaging/installer/kickstart.sh

```diff
@@ -772,15 +772,64 @@ claim() {
   fi
 
   # shellcheck disable=SC2086
-  if ${ROOTCMD} "${NETDATA_CLAIM_PATH}" -token="${NETDATA_CLAIM_TOKEN}" -rooms="${NETDATA_CLAIM_ROOMS}" -url="${NETDATA_CLAIM_URL}" ${NETDATA_CLAIM_EXTRA}; then
-    progress "Successfully claimed node"
-  else
-    warning "Unable to claim node, you must do so manually."
-    if [ -z "${NETDATA_NEW_INSTALL}" ]; then
-      cleanup
-      trap - EXIT
-      exit 1
-    fi
+  ${ROOTCMD} "${NETDATA_CLAIM_PATH}" -token="${NETDATA_CLAIM_TOKEN}" -rooms="${NETDATA_CLAIM_ROOMS}" -url="${NETDATA_CLAIM_URL}" ${NETDATA_CLAIM_EXTRA}
+  case $? in
+    0)
+      progress "Successfully claimed node"
+      return 0
+      ;;
+    1)
+      warning "Unable to claim node due to invalid claiming options. If you are seeing this message, you’ve probably found a bug."
+      ;;
+    2)
+      warning "Unable to claim node due to issues creating the claiming directory or preparing the local claiming key. Make sure you have a working openssl command and that ${INSTALL_PREFIX}/var/lib/netdata/cloud.d exists, then try again."
+      ;;
+    3)
+      warning "Unable to claim node due to missing dependencies. Usually this means that the Netdata Agent was built without support for Netdata Cloud."
```

Comment:
Yeah, this needs improved, but it’s not just a direct docs link, because we want them to report a bug if they get this for native packages (or a static build, but we actually _do_ have a couple of bugs there that can cause this case that need fixed).

Reply:
Made a suggestion below.